porcupine/plugins/run/no_terminal.py

```diff
@@ -477,3 +477,16 @@ def run_command(command: str, cwd: Path) -> None:
     assert runner is not None
     get_vertical_panedwindow().paneconfigure(runner.textwidget, hide=False)
     runner.run_command(cwd, command)
+    # Use previous height as long as it's not less than two lines high,
+    # otherwise set height to two lines high.
+    current_height = textutils.textwidget_size(runner.textwidget)[1]
+    padding = textutils.get_padding(runner.textwidget)[0]
```

Comment:
We want the y padding:
```suggestion
    padding = textutils.get_padding(runner.textwidget)[1]
```